mk/containerized.mk

```
@@ -19,7 +19,12 @@ pull: ## Download SVC images
 build:  ## Build SVC images
 build: DOCKER_COMPOSE_ENV = DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
 
-push: ## Push SVC container images to a registry. Requires previous "docker login"
+GIT_LATEST_TAG := $(shell git tag --sort=taggerdate | tail -1)
```

`!=` automatically sets a variable to the output of the shell command.
```suggestion
GIT_LATEST_TAG != git tag --sort=taggerdate | tail -1
```

Additionally, let's rather use `git describe`, as it specifically relates a given commit to a named reference (tag).
```suggestion
GIT_LATEST_TAG != git describe --tag --abbrev=0
```

That said, this approach wouldn't be fully working, as it needs to update also:
* `.env` `NVMEOF_VERSION`
* `pyproject.toml` `version`.

Since that's going to be a challenging issue, I'd leave it for a later PR.

---

@epuertat , the command `:= $(shell git tag --sort=taggerdate | tail -1)` works as expected, your propossed command brings the first created version and not the latest one.

---

The fact that `git describe` is giving you an older version is the proof that it's working, while the `git tag` doesn't.

Tags can be assigned to any branch (and can be re-pushed, so the `taggerdate` would also be problematic), so `git tag` will return the latest tag irrespective of whether the branch you're building and pushing contains that tag or not.

`git describe` tells you exactly what's the latest tag a specific commit descends from. There's no other way to do this correctly.

---

done